data/json/monstergroups/zombies.json

```diff
@@ -531,7 +547,9 @@
     "monsters": [
       { "monster": "mon_feral_human_pipe", "freq": 100, "cost_multiplier": 1, "pack_size": [ 3, 8 ] },
       { "monster": "mon_feral_human_crowbar", "freq": 40, "cost_multiplier": 1, "pack_size": [ 2, 6 ] },
-      { "monster": "mon_feral_human_axe", "freq": 20, "cost_multiplier": 2, "pack_size": [ 1, 4 ] }
+      { "monster": "mon_feral_human_axe", "freq": 20, "cost_multiplier": 2, "pack_size": [ 1, 4 ] },
+      { "monster": "mon_feral_prepper", "freq": 1, "cost_multiplier": 25 },
```

Comment:
I like it having a high multiplier but having experience with having jacked up the multiplier of Headless Horror's previously I can tell you that the player base will more likely view it as "all monsters are now feral gunmen!", despite the fact that every feral gunman is replacing 25 other ferals and zombies.

Reply:
I just realized the Frequency is 1.  That should be low enough to stop the complaints I foresaw,

Reply:
Phew, is there anything I should do or edit in the monster group frequency you may suggest?